packages/eddsa-proof/src/generate.ts

```diff
@@ -1,5 +1,7 @@
 import { deriveSecretScalar } from "@zk-kit/eddsa-poseidon"
-import { SnarkArtifacts, maybeGetEdDSASnarkArtifacts, packGroth16Proof } from "@zk-kit/utils"
+import type { SnarkArtifacts } from "@zk-kit/utils"
```

Comment:
btw if you care about enforcing and automating sorting imports (and object properties), there is 
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/newline-after-import.md
https://github.com/mskelton/eslint-plugin-sort/tree/main

Reply:
Ahhh interesting! I use a nvim shortcut to sort them. I think that is also based on eslint actually.